packages/terra-application-navigation/src/tabs/_Tabs.jsx

```diff
@@ -139,33 +143,16 @@ class Tabs extends React.Component {
   }
 
   resetCalculations() {
+    this.animationFrameID = null;
     this.hiddenStartIndex = -1;
     this.menuHidden = false;
     this.isCalculating = true;
   }
 
-  initializeResize() {
-    if (!this.resizeListenerAdded) {
-      this.resizeObserver = new ResizeObserver((entries) => {
-        this.contentWidth = entries[0].contentRect.width;
-        this.updateSize();
-      });
-      this.resizeObserver.observe(this.containerRef.current);
-      this.resizeListenerAdded = true;
-    }
-  }
-
-  removeResize() {
-    if (this.resizeListenerAdded) {
-      this.resizeObserver.disconnect(this.containerRef.current);
-      this.resizeListenerAdded = false;
-    }
-  }
-
   handleResize(width) {
     // Calculate hide index
     const childrenCount = this.props.navigationItems.length;
-    const moreWidth = width - this.getRollupTabWidth();
+    const moreWidth = this.rollupTabRef.current ? width - this.getRollupTabWidth() : width;
```

Comment:
Instead of inlining it here I might make more sense to put this check in the `getRollupTabWidth` method, where it would return zero if it wasn't present.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1160/commits/16a7e89d88caf49e801c8cf21fab8dceeaad2f09.